mobile/lib/src/domain/settings/settings.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:equatable/equatable.dart';
+
+part 'sex.dart';
+
+class Settings extends Equatable {
+  final int age;
+  final String serverAddress;
+  final Sex sex;
+
+  Settings({this.age, this.serverAddress, this.sex});
```

Comment:
Perform validation according to our business logic rules on object construction.

Reply:
assert(age != null), assert(serverAddress != null, assert(sex != null)

Voici la valdation que nous faisions côté front-end
12 <= age <= 125
serverAddress doit avoir la forme d'une addresse IP.
Il n'y a pas vraiment de validation à faire sur sex comme c'est un enum et qu'on fait déjà le null check.